packages/io/x3d-serializer/src/index.js

```diff
@@ -188,6 +190,7 @@ const convertGeom3 = (object, options) => {
   const shape = ['Shape', {}, convertMesh(object, options)]
   if (object.color) {
     shape.push(convertAppearance(object, options))
+    shape[3][1][1].emissiveColor = '0 0 0'
```

Comment:
Anyway to make this more intuitive? Maybe convertMesh can add emissiveColor?

Reply:
The idea would be to add emissiveColor for Lines, and only diffuseColor for Meshes. I would like to add a parameter to convertAppearance() . Let me update if this sounds acceptable.